tesseract-core/src/lib.rs

```diff
@@ -202,9 +283,14 @@ impl Schema {
         let table = self.cube_table(&cube)
             .ok_or(format_err!("No table found for cube {}", cube))?;
 
-        let cut_cols = self.cube_cut_cols(&cube, &query.cuts)
+        let mut cut_cols = self.cube_cut_cols(&cube, &query.cuts)
+            .map_err(|err| format_err!("Error getting cut cols: {}", err))?;
+
+        let default_hierarchy_cut_cols = self.cube_cut_cols(&cube, &default_hierarchy_cuts_query)
```

Comment:
A small thing, but maybe mention in the error message that this failed for the default hierarchy cut cols, instead of the regular cut cols.

Reply:
good point